client/state/ui/guided-tours/test/fixtures/config.js

```diff
@@ -21,7 +21,7 @@ export const MainTour = makeTour(
 );
 
 export const ThemesTour = makeTour(
-	<Tour name="themes" version="test" path="/design" when={ () => true } >
+	<Tour name="themes" version="test" path="/themes" when={ () => true } >
```

Comment:
Creating a function on each render seems the only hard linting error in he build (since this line was changed, it was enforced). Fixing it seems easier than ignoring the linting results.